<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="9">
      <widget class="QSpinBox" name="spinBoxVelocity">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gravity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="16">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGravity">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="value">
        <double>-9.800000000000001</double>
       </property>
      </widget>
     </item>
     <item row="0" column="14">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Time scale</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="10">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="horizontalSliderPosition">
       <property name="maximum">
        <number>720</number>
       </property>
       <property name="value">
        <number>380</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="15">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTimescale">
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="0" column="17">
      <widget class="QPushButton" name="pushButtonLauch">
       <property name="text">
        <string>Launch</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Velocity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSlider" name="horizontalSliderAngle">
       <property name="toolTipDuration">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="17">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="renderHints">
        <set>QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
       <property name="optimizationFlags">
        <set>QGraphicsView::DontAdjustForAntialiasing|QGraphicsView::DontClipPainter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="13">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderAngle</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
